src/test/java/org/redischool/attendance/details/CourseDetailsRepositoryTest.java

```diff
@@ -125,8 +125,8 @@ public class CourseDetailsRepositoryTest {
 
             describe("update course", () -> {
                 it("doesn't do anything with empty update data", () -> {
-                    courseDetailsRepository.updateAttendance("1", "", new HashMap<String, String>());
-                    verifyZeroInteractions(googleSheetsApi);
+                    courseDetailsRepository.updateAttendance("1", "", new HashMap<>());
+                    verifyNoMoreInteractions(googleSheetsApi);
```

Comment:
That's not true, it's changing the meaning of the assertion. Before we were making sure we didn't interact with GoogleSheetsApi at all. Now we're theoretically allowing some interactions and then making sure nothing else has happened. What we want are zero interactions, please revert.

Reply:
That was a mistake when I replaced the deprecated verifyZeroInteractions, I updated it now to verifyNoInteractions.